lib/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/guardians_bloc.dart

```diff
@@ -59,22 +60,22 @@ class GuardiansBloc extends Bloc<GuardiansEvent, GuardiansState> {
 
   FutureOr<void> _initial(Initial event, Emitter<GuardiansState> emit) async {
     emit(state.copyWith(pageState: PageState.loading));
-    final result = await _getGuardiansDataUseCase.getGuardiansData();
+    final Result<GuardiansConfigModel> result = await _getGuardiansDataUseCase.getGuardiansData();
 
-    // TODO(n13): Should use mapper
-    final guardiansModel = (result.isValue && result.asValue!.value != GuardiansConfigModel.empty)
```

Comment:
this is not needed, the service returns an error or a model. NO need to create another model

Reply:
But... the initial state when there is no model on the server should be empty model, no? So we can start filling it up...